Unix/nits/injector/injector.cpp.bkp

```diff
@@ -126,24 +126,14 @@ PWSTR ConvertStringToW(_In_opt_z_ const char *buf)
 
```

Comment:
Again, shouldn't have backup copies.  Delete.

Reply:
deleted it.